samples/sample_common/src/sample_utils.cpp

```diff
@@ -196,19 +203,61 @@ mfxStatus CSmplYUVReader::LoadNextFrame(mfxFrameSurface1* pSurface)
             break;
         case MFX_FOURCC_YUY2:
             pitch = pData.Pitch;
-            ptr = pData.Y + pInfo.CropX + pInfo.CropY * pData.Pitch;
+            ptr = pData.Y + pInfo.CropX*2 + pInfo.CropY * pData.Pitch;
```

Comment:
This was not in prev. version.... This is a separate bug fix.

Reply:
It was... you might be missing it, but this issue was found during applying new  color formats